parm/config/config.vrfy

```diff
@@ -53,25 +53,25 @@ export RUN_MAPSDA="NO"
 #model_hpss_dir_list:    HPSS directory of data
 #model_data_run_hpss:    get files from HPSS ("YES") if not online
 #hpss_walltime:          how long to run HPSS job to get data in minutes
-export model_list="gfs"
-export model_dir_list="/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/global/archive/model_data"
-export model_stat_dir_list="/lfs/h2/emc/vpppg/noscrub/$USER/archive"
-export model_file_format_list="pgbf{lead?fmt=%2H}.gfs.{init?fmt=%Y%m%d%H}"
+export model_list="C96_TEST_NEW"
+export model_dir_list="/gpfs/f5/epic/scratch/$USER/archive"
+export model_stat_dir_list="/gpfs/f5/epic/scratch/$USER/archive"
+export model_file_format_list="pgbf{lead?fmt=%2H}.gfs.{init?fmt=%Y%m%d%H}.grib2"
 export model_data_run_hpss="NO"
 export model_hpss_dir_list="/NCEPPROD/hpssprod/runhistory"
 export hpss_walltime="10"
 ## OUTPUT DATA SETTINGS
 #OUTPUTROOT: base output directory
-export OUTPUTROOT="/lfs/h2/emc/stmp/$USER/verif_global_standalone"
+export OUTPUTROOT="/gpfs/f5/epic/scratch/$USER/verif_global_standalone"
 ## DATE SETTINGS
 #start_date:          verification start date, format YYYYMMDD
 #end_date:            verification end date, format YYYYMMDD
 #spinup_period_start: spinup period start, format YYYYMMDDHH, if none use "NA"
 #spinup_period_end:   spinup period end, format YYYYMMDDHH, if none use "NA"
 #make_met_data_by:    how to treat dates, "VALID" or "INIT"
 #plot_by:             how to plot data, "VALID" or "INIT"
-export start_date=$(date -d "24 hours ago" '+%Y%m%d')
-export end_date=$(date -d "24 hours ago" '+%Y%m%d')
+export start_date=20210323
+export end_date=20210323
```

Comment:
These changes are fine for testing, but should probably be reverted when complete.

Reply:
Please revert the start/end date.